src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverrideReporterTest.java

```diff
@@ -0,0 +1,26 @@
+package edu.hawaii.its.groupings.configuration;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
```

Comment:
Can you use the hamcrest assertThat here instead? We've been using that everywhere else.

Reply:
done.